receiver/splunkhecrawreceiver/README.md

```diff
@@ -0,0 +1,45 @@
+# Splunk HEC Raw Receiver
+
+The Splunk HEC raw receiver accepts data delimited by newline separators and ingests them as logs.
+
+Supported pipeline types: logs
+
+> :construction: This receiver is in beta and configuration fields are subject to change.
+
+## Configuration
+
+The following settings are required:
+
+* `endpoint` (default = `0.0.0.0:8088`): Address and port that the Splunk HEC
+  receiver should bind to.
+
+The following settings are optional:
+
+* `access_token_passthrough` (default = `false`): Whether to preserve incoming
+  access token (`Splunk` header value) as
+  `"com.splunk.hec.access_token"` metric resource label.  Can be used in
+  tandem with identical configuration option for [Splunk HEC
+  exporter](../../exporter/splunkhecexporter/README.md) to preserve datapoint
+  origin.
+* `tls_settings` (no default): This is an optional object used to specify if TLS should be used for
+  incoming connections.
+    * `cert_file`: Specifies the certificate file to use for TLS connection.
+      Note: Both `key_file` and `cert_file` are required for TLS connection.
+    * `key_file`: Specifies the key file to use for TLS connection. Note: Both
+      `key_file` and `cert_file` are required for TLS connection.
+* `path` (default = '/*'): The path to listen on, as a glob expression.
```

Comment:
This config is surprising to me - why isn't the path a fixed value? Or if it doesn't really matter, why do we need to allow a configurable glob instead of accepting any?

Reply:
It doesn't really matter. It's lifted from splunkhecreceiver which has the same config. There was some thinking to be able to combine servers so the same port could serve both, but it's probably not going to happen this way.

Reply:
Yeah figured a lot was copied in. I think we can do some cleanups, this path doesn't seem to fit well with the collector model (maybe it should, but for now the collector exposes different services on ports, not paths like the splunk one does from what I can tell). So it'd be good to remove it here

Reply:
done, removed.